src/web/views/send-code.njk

```diff
@@ -4,29 +4,29 @@
 
 {% block formContent %}
     {{ govukRadios({
-        id: "choose-channel-for-code",
-        name: "chooseChannelForCode",
+        id: "channel-for-code",
```

Comment:
I think `method` rather than channel would work better in this file too.

Reply:
I have no real strong feelings either way on this. @gethyn1 @YetAnotherMatt opinions?

Reply:
I'm happy with channel. It's purely internal at this point and the difference isn't great.